pkg/operator/ceph/nfs/security.go

```diff
@@ -225,7 +247,7 @@ cat /tmp/etc/krb5.conf`,
 		Resources: nfs.Spec.Server.Resources,
 	}
 
-	return init, volume, ganeshaMount
+	return
```

Comment:
While unnecessary in Go, the Rook project prefers to keep returns with full value spec so that returns are  easier to read without looking at the fxn signature

Reply:
I'll change this one.